packages/components/organisms/f-loyalty/src/components/Header.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+    <div :class="$style['c-loyaltyHeader']">
+        <div :class="$style['c-loyaltyHeader-container']">
+            <bread-crumbs
+                :links="links"
+                :router-links="true" />
+            <media-element
+                :class="$style['c-loyaltyHeader-media']"
+                :text-size="fontSize"
+                :title="$t('header.title')"
+                :text="$t('header.text')"
+                :image-url="getImageBasedOnScreenSize"
+                image-align="center"
+                :flex="flexLayout"
+                :alt="$t('header.alt')"
+            />
+        </div>
+    </div>
+</template>
+
+<script>
+import BreadCrumbs from '@justeat/f-breadcrumbs';
+import MediaElement from '@justeat/f-media-element';
+import '@justeat/f-media-element/dist/f-media-element.css';
+import '@justeat/f-breadcrumbs/dist/f-breadcrumbs.css';
+import debounce from 'lodash.debounce';
+import { FONT_SIZE } from '@justeat/f-media-element/src/constants';
```

Comment:
This seems strange that you need to import in the font_size to then pass it back through to the mediaElement component above.

Doesn't need to be changed necessarily as part of this PR, but I would probably provide an abstraction so that `mediaElement` can take in a prop identifier (like `small/medium/large` for the fontSize), and then the media element just links that ID to the actual value as part of the component?

Reply:
Just had another look at this – it does look like this prop is a string already?

If so, I'd probably just pass through the string rather than import it from the component and then pass it back through. For me, it's expected that consumers of the component would need to know how to define the props (may be worth a second opinion, but I think just passing in `xl` is fine for this?)

Reply:
your correct its my mistake, just used to typing that constant. Its not correct as I am not exporting those constants as part of f-media-element package.

Reply:
I will make the change